src/Reports/SiteSummary.php

```diff
@@ -25,7 +27,14 @@ public function description()
 
     public function sourceRecords($params)
     {
-        return Package::get()->filter('Type:StartsWith', 'silverstripe');
+        $packageList = Package::get();
+        $typeFilters = Config::inst()->get(UpdatePackageInfo::class, 'allowed_types');
+
+        if (!empty($typeFilters)) {
+            $packageList = $packageList->filter('Type', $typeFilters);
+        }
+
```

Comment:
Can you please add an extension to update the package list here? Then I can remove the same change from #56